include/onnx-mlir/Runtime/OMTensor.h

```diff
@@ -52,14 +52,6 @@ struct OMTensor;
 typedef struct OMTensor OMTensor;
 #endif
 
-/* Helper function to compute the number of data elements */
```

Comment:
Did you just move the code below? Or was there a substantive change, I could not tell.

Reply:
Just moved the helper function `getNumOfElems` from user header into `OMTensor.inc` since it really is an internal function not meant for the user.